### ISSUE TITLE
About homeDataDir

### ISSUE THREAD

**Issue by hope366** [lightvector/KataGo#249]
The changes in katago-v1.4.4, which are
For GTP and other commands, you can now specify homeDataDir=<DIR> to override the directory where KataGo will cache some data (currently, the OpenCL tuner files on the OpenCL version).
Questions about.
homeDataDir=### and set the opcl version of katago to When you run it, the opcl tuning file is saved in the #### folder, but you can use The KataGoData folder is now just a #### folder, but I can't see the difference otherwise! 
What exactly does "it is now possible to specify to override" mean? Is it?

**Comment by lightvector** [comment]
I'm sorry, I don't understand what you are asking. homeDataDir is the directory inside of which a directory to hold OpenCL tuning files will be created. If homeDataDir is a relative path, then it will be relative to your current working directory when running it, and if it is an absolute path, then it will be that absolute path. If it is unspecified, it will be "DIR/KataGoData" where DIR is the directory containing the katago.exe.

**Comment by hope366** [comment]
Thank you for your answer.
I don't know if there's a problem with my wording or with the translation function, but I'll try to write it again carefully.
In katago-v1.4.2 and earlier, run the opcl version of katago The OpenCL tuning file will be created in "DIR / KataGoData". 

If you run katago again, the tuning will be cut and the startup time will be reduced a bit. The KataGoData directory remains intact and nothing else is generated.

In katago-v1.4.4, when the KataGoData directory exists, you can use If you specify homeDataDir, you can use homeDataDir The OpenCL tuning file will be created in the directory specified in
In other words, if you rewrite homeDataDir, a new directory will be created every time you specify it.
Is this the change?
When you say "you can now specify to override" do you mean this?

**Comment by lightvector** [comment]
The purpose of homeDataDir option is to be able to store the OpenCL tuning files in a different place besides the directory containing the KataGo executable, such as if the KataGo executable's directory is non-writeable.

If you change it, of course then at the new location the tuning file won't exist yet, so it will need to tune again to generate the file. But then after that, if you keep homeDataDir the same, it should continue to use that new location.

**Comment by hope366** [comment]
I hadn't thought about "if it's not writable" at all.
Certainly if you consider such a case. I think this added feature is a useful one.
You've helped me understand a lot about homeDataDir.
Thank you very much.
